ui/ui-components/types/app.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
Perhaps this shouldn't be called `app` (which might be confused with an `App`).  Perhaps "webApp" or "UI"?